File: server.bgst/Requests/GetRequests/SearchRequest.cs
namespace server.bgst.Requests.GetRequests;

public class SearchRequest 
{
    public int Page { get; set; }
    public int PageCount { get; set; }
    public string Name {get; set;} = "";
    public int MinPlayers { get; set; }
    public int MaxPlayers { get; set; }
    public int MinPlayTime { get; set; }
    public int MaxPlayTime { get; set; }
}

File: client.bgst/src/features/board-game/types/search-request.ts
export interface SearchRequest {
    page: number,
    pageCount: number,
    name: string,
    minPlayers: number,
    maxPlayers: number,
    minPlaytime: number,
    maxPlaytime: number,
    order: number
}

export interface SearchRequestController{
    searchRequest: SearchRequest
    setSearchRequest: React.Dispatch<React.SetStateAction<SearchRequest>>
}